_docs-2-0/administration/upgrading.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Upgrading Accumulo
+category: administration
```

Comment:
Nice. I think it's a good idea this in the documentation. It can be linked to from release notes but it's nice that this will carry forward in the docs with each release for historical purposes.

Reply:
Should the upgrade documentation exist in this centralized docs, or should we focus more on upgrade steps in the release notes, as they apply to that specific release? Either way, there should be a mention in both places. I'm just curious where the details should go. Personally, I like them in the release notes more. I'm just worried that the details will end up spreading across both locations over time.

Reply:
I can understand the preference for release notes, but the conversation started when a user asked how to upgrade from 1.4 to 1.8 and there was some uncertainty as to how easy it was for users to find the upgrade information, especially across multiple version jumps.  The thought was that if there was one central location where the information resided then it would be easier to point users to it and it would also serve as a historical record of upgrade processes. Once the 2.0 docs are 'released' we could update release notes to link to the proper information on this page and allow it to be the official location for upgrade guidance.

Reply:
> Once the 2.0 docs are 'released' we could update release notes to link to the 

We may also want to consider making the readme.md link to the upgrade docs (instead of having upgrading.md).

Reply:
@jmark99 Understood. I'm thinking ahead, to the situation where the 2.0 upgrade section of the docs doesn't fully describe a situation for 2.0.9, because of some bug which specifically affected the 2.0.8 to 2.0.9 upgrade (as an example). Would this get added to the main 2.0 docs, or just called out in the 2.0.9 release notes? Or both? Are we prepared to put all patch-specific upgrade notes in the major-minor release notes, or would we want to describe the overall process, and mention "check the release notes for your version, for any upgrade issues specifically for that version"?

I don't feel strongly about one way or another. I just want to have a plan in place for when we encounter this scenario.

Reply:
I see your point. Given the option, I would lean toward having the overall process described in the upgrade documentation with the stipulation that a user should check the release notes for any specific issues for that particular version. But I'm interested to hear what others would prefer as well.

Reply:
We could even suggest checking the release notes in the user manual upgrade docs.

Reply:
@keith-turner Wait, I'm confused now. I thought that's what we were talking about.